src/core/Geocoding.js

```diff
@@ -114,16 +114,20 @@ module.exports = function Geocoding(options) {
         async: false,
         url: url
     });
-    onComplete = onComplete || function onComplete(geometry) {
-      $("#lat").val(geometry.lat);
-      $("#lng").val(geometry.lng);
-
-      map.setView([geometry.lat, geometry.lng], options.zoom);
+    onComplete = onComplete || function onComplete(response) {
+      if(response.status === "OK") {
```

Comment:
Hey @nstjean , awesome!
Do we have response.code === 200? Thanks!

Reply:
Oh you mean in addition to the response.status from google we should also check the http response code?

Reply:
Yes usually i have seen people checking code but i guess status also works fine!